packages/babel-parser/src/parser/expression.js

```diff
@@ -33,14 +33,18 @@ import * as charCodes from "charcodes";
 import {
   BIND_OUTSIDE,
   BIND_VAR,
-  functionFlags,
   SCOPE_ARROW,
   SCOPE_CLASS,
   SCOPE_DIRECT_SUPER,
+  SCOPE_FUNCTION,
   SCOPE_SUPER,
   SCOPE_PROGRAM,
-  SCOPE_ASYNC,
 } from "../util/scopeflags";
+import {
+  PARAM_AWAIT,
+  PARAM_,
```

Comment:
Nit: what about `PARAM_DEFAULT`, or just `PARAM`? The trailing underscore looks... weird.

Reply:
Well the spec does have `[Default]` parameter so `PARAM_DEFAULT` will be confusing as we don't check `[Default]` yet.

I have come up with names like `PARAM_NONE`, `PARAM_OTHER` when authoring but I am afraid they maybe used at the spec one day.

I will go with `PARAM` then.